code/game/objects/items/devices/radio/headset.dm

```diff
@@ -230,6 +230,14 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 	icon_state = "srv_headset"
 	keyslot = new /obj/item/encryptionkey/headset_service
 
+/obj/item/radio/headset/headset_clown
+	name = "clown's headset"
+	desc = "A headset for the clown. Finally. A megaphone you can't take away."

```

Comment:
grammar check grammar check

**Finally, a megaphone you can't take away.** would flow better imo

Reply:
screee I purposefully put it like that